reana_commons/config.py

```diff
@@ -349,15 +349,8 @@ def workspaces(workspaces_list):
 WORKSPACE_PATHS = workspaces(os.getenv("WORKSPACE_PATHS"))
 """List of allowed workspace paths."""
 
-def mounted_paths(mounted_paths):
-    mounted_list = mounted_paths.split(",") if mounted_paths else []
-    mounted_paths= {}
-    for path in mounted_list:
-        mount = path.split(":")
-        mounted_paths[mount[0]]=mount[1]
-    return mounted_paths
-    
-MOUNTED_PATHS = mounted_paths(os.getenv("MOUNTED_PATHS"))
+MOUNTED_PATHS = os.getenv("MOUNTED_PATHS", None)
```

Comment:
Please have a look into [my comment](https://github.com/reanahub/reana/pull/538/files#r709148381). I think instead of introducing a new config variable here, we could store a dict of paths like you did in `extract_mounted_paths()` in WORKSPACE_PATHS config. We could use it for validation and as well for setting the right `hostPath` to `reana-workspace-volume`. We can set a `DEFAULT_WORKSPACE_PATH` config as well from this dict, doing all the manipulations here in python would be better than in helm. WDYT?